src/game/mod.rs

```diff
@@ -350,7 +350,28 @@ where
     G: SimpleSum<N>,
 {
     fn utility(&self, state: State) -> [Utility; N] {
-        todo!()
+        SimpleSum::utility(self, state).map(|x| x as Utility)
+    }
+}
+
+impl<G> SimpleSum<2> for G
+where
+    G: ClassicGame + Extensive<2>,
```

Comment:
Let's change `ClassicGame` to require `Extensive<2>`  so that this line only reads `G: ClassicGame`.